README.md

```diff
@@ -473,6 +473,40 @@ operators:
 See [jitdb operators] and [operators/index.js] for a complete list of supported
 operators.
 
+### decrypted
+
+A pull-stream source with newly decrypted values returned as full
```

Comment:
The implementation does something slightly different, it only emits decrypted values from reindexEncrypted, not all the other conventional decryptions.

I have two ideas:

1. Rename this to reflect that it is only for reindexEncrypted decryptions. I'm also worried that the name `decrypted` is too similar to `operators.js` / `decrypted`
2. What if `reindexEncrypted` *returned* a pull-stream of all the decryptions that occurred in that specific call?

I like option 2, but I'm not sure if it's going to be more developer friendly or less developer friendly.

Reply:
The name change is a good idea. About option 2, I'm afraid that might lead to code where you in multiple places would have to handle there values returned. Instead with the stream approach you can more easily combine there where you handle messages of a specific type anyway I think.